.changeset/healthy-shrimps-notice.md

```diff
@@ -0,0 +1,5 @@
+---
+'@backstage/plugin-scaffolder-backend': minor
+---
+
+Added `requireConversationResolution` template option to `github:repo:create`, `github:repo:push` and `publish:github`
```

Comment:
```suggestion
Added `requiredConversationResolution` template option to `github:repo:create`, `github:repo:push` and `publish:github`
```

Let's keep the name you first suggested :)

Reply:
Done.